File: cpp-programs/class-objects/returning-object.cpp
#include<string.h>
#include<bits/stdc++.h>
using namespace std;
#define PI 3.14

class Circle
{
 float radius;
 public:
 
 void setradius();
 float getradius();
 void showradius();

};


void Circle :: setradius()
{
  cout<<"Enter radius :::";
  cin>>radius;
}


float Circle :: getradius()
{
  return radius;
}

Circle createcircle()
{
 Circle c;
 c.setradius();
 return c;
}

 
float calarea(Circle c)
{
  float ar;
  ar = PI * c.getradius() * c.getradius() ;
  return ar;
}

void Circle :: showradius()
{
  cout<<"Radius : "<<radius<<endl;
}
 
 
 
int main()
{
 Circle c1 = createcircle();
 
 c1.showradius();
 
 float a = calarea(c1);
 cout<<"Area of circle : "<<a<<endl;
  
}

File: java-programs/class-objects/returning-object.java
import java.io.*;
import java.util.Scanner;


class circle
{
  private double radius;
  
  circle()
  {
    radius = 0.0;
  }

  circle(double radius)
  {
    this.radius = radius;
  }

  public double area()
  {
    return 3.14 * radius * radius;
  }
}


class returning_object
{
/**Main function**/
 public static void main(String args[])
{
    circle c1 ;
    c1 = createcircle();
    calcarae(c1);
}
public static circle createcircle()
{
    circle c = new circle(10);
    return c;
}

public static void calcarae(circle c)
{
    System.out.println("Area of circle : "+c.area());
}

}

